src/js/components/AppPageControlsComponent.jsx

```diff
@@ -68,7 +68,7 @@ var AppPageControlsComponent = React.createClass({
             onClick={this.handleScaleApp}>
           Scale Application
         </button>
-        <button className="btn btn-lg btn-default"
+        <button className={"btn btn-lg btn-default " + suspendAppClassSet}
```

Comment:
I would prefer to have a specific class set for the restart button. Otherwise we may get unforeseen side effects if we ever change the `suspendAppClassSet`

This is what i mean:
```JS
var restartAppClassSet = classNames("btn btn-lg btn-default ", {
    "disabled": props.model.instances < 1
});
```

Otherwise I do like the idea. Thanks for this Pull Request. If you add this change I will make sure it gets merged into master asap.